sympy/discrete/transforms.py

```diff
@@ -0,0 +1,69 @@
+"""
+Discrete Fourier Transform, Walsh Hadamard Transform,
+Number Theoretic Transform, Zeta Transform, Mobius Transform
+"""
+from __future__ import print_function, division
+
+from sympy.core import S, Symbol, sympify
+from sympy.core.compatibility import as_int, range, iterable
+from sympy.core.function import expand, expand_mul
+from sympy.core.numbers import pi, I
+from sympy.functions.elementary.exponential import exp
+from sympy.ntheory import isprime, primitive_root
+from sympy.utilities.iterables import ibin
+
+def _fourier_transform(seq, inverse=False):
+    """
+    Performs the DFT in complex field, by padding zeros as
+    radix 2 FFT, requires the number of sample points to be
+    a power of 2.
```

Comment:
I see, you are saying here that it does do the padding. The incorrect comma after "FFT" confused me. 

I think this would be clearer

```
Performs the discrete Fourier transform (DFT) in the complex domain

The sequence is automatically padded to the right with zeros, as the radix 2 FFT requires the number of sample points to be a power of 2.
```

Also this would be better put in the public API functions.

Reply:
Sure. Will update it.